src/managers/FileManager/useFileManagerConfig.js

```diff
@@ -284,8 +307,17 @@ export function useFileManagerConfig() {
 		};
 	}
 
-	function getColumns() {
+	function getColumns({managerConfig}) {
 		const columns = [];
+		const enabledActions = managerConfig.permittedActions;
+
+		if (enabledActions.includes(Actions.FILE_SELECT)) {
+			columns.push({
+				header: t('common.numero'),
```

Comment:
Can we use "Select Files" as the header instead? 
Also needs `headerSrOnly: true,` like in "More Actions" column.
```suggestion
				header: t('editor.submission.selectFiles'),
				headerSrOnly: true,
```
I'll leave it to you if we should reuse `editor.submission.selectFiles` or create a `common.selectFiles` key.